sachungo/movies
# Search by movie name, not actor

### Which issue does this PR relate to?

N/A

### What are the changes implemented in the PR?



Change the search to be by `movie` not `actor` name

### How should the implementation be QA'd?



- Verify that the search placeholder says `Search by movie name`
- Verify that the search request returns a list of movies

### Anything else to note


N/A

### Screenshots and/or gifs


N/A
